build-system-tests/templates/components/react/cra/App.js

```diff
@@ -1,22 +1,29 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
+import { Marker } from 'react-map-gl';
 import { Amplify, Notifications } from 'aws-amplify';
 import {
   AccountSettings,
   Authenticator,
   FileUploader,
   MapView,
```

Comment:
`MapView` should be imported from `@aws-amplify/ui-react-geo`

Reply:
Good catch!